### DISCUSSION TITLE
Name

### DISCUSSION THREAD

**Discussion by sayanarijit** [ratatui/ratatui#72] [Polls]
Since the author of the original crate is not responding, we have decided to go with a new name after all. The intention is to eventually merge the changes back to the original crate if the author comes back and finds the changes acceptable. However, it is also possible that the revived crate could end up being a successor or an alternative to the original crate ([more info](https://github.com/tui-rs-revival/tui-rs-revival/discussions/70)).

In any case, we have collected many naming suggestions and decided to release the crate under a new name from the following list: https://docs.google.com/spreadsheets/d/1_9C9mYKFhN6ssW0Bg9aWjRsrdvKeik7WHS1j1W5e0bw/edit?usp=sharing

Because the GitHub discussions poll is limited to only 8 options, we will be using the comment section and emojis as a better alternative.

The poll will be run for 24 hrs, and can be extended upto 48 hrs if the community thinks it should.

----------

## Update

We have a clear winner for the name and version for the next release. The next release will be:

```toml
ratatui = "0.20.0"
```

**Comment by sayanarijit** [comment]
ratatui

**Reply by nidnogg** [reply]
I skimmed over this one and initially dismissed it but after seeing it blast this poll I'm now a believer

**Reply by Owletti** [reply]
Anyone can code, but only the fearless can be great.

\- Ratatui

**Reply by foodornt** [reply]
Ratatouille

**Comment by sayanarijit** [comment]
tuint

**Comment by sayanarijit** [comment]
intuition

**Comment by sayanarijit** [comment]
noctuid (night flying moth)

**Comment by sayanarijit** [comment]
tuille (used for decoration)

**Comment by sayanarijit** [comment]
etui (a container, squatted)

**Comment by sayanarijit** [comment]
lrtl (lightweight rust tui library)

**Comment by sayanarijit** [comment]
intuit (squatted)

**Comment by sayanarijit** [comment]
ptui (spitting sound)

**Comment by sayanarijit** [comment]
tea-ui

**Comment by sayanarijit** [comment]
teaui

**Comment by sayanarijit** [comment]
rtl (rust tui library)

**Comment by sayanarijit** [comment]
noctuid

**Comment by sayanarijit** [comment]
two-e

**Comment by sayanarijit** [comment]
mark

**Comment by sayanarijit** [comment]
tutu

**Comment by sayanarijit** [comment]
tui2

**Comment by sayanarijit** [comment]
tuile

**Comment by sayanarijit** [comment]
tui-2e

**Comment by niurenyige** [comment]
atui(another tui)

**Reply by conradludgate** [reply]
(I am biased as a maintainer) this is a bit too close to `atuin`, a TUI app written in Rust.

**Comment by Igonato** [comment]
Hi there! Just wanted to throw in an idea, how do you feel about **"Rust's Amazing Terminal Agnostic Text User Interfaces"** for a backronym?

Terminal agnostic - referring to it supporting multiple backends already, and I can see it having a web support in the future (I have free time and would like to put some of my hours towards it, but just recently discovered the project and haven't dived in the code yet, so putting in some hours is all I can promise).

**Reply by mindoodoo** [reply]
I love the backronym man !

If you haven't joined our discord yet, you definitely should ! I believe some of the server members were actually talking about building a new backend to enable terminal like uis in the browser.
